content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -159,7 +159,7 @@ Feature | Default | Alpha | Beta | GA
 :-------|:--------|:------|:-----|:----
 `PublicKeysECDSA` | `false` | 1.19 | - | -
 `RootlessControlPlane` | `false` | 1.22 | - | -
-`UnversionedKubeletConfigMap` | `true` | 1.22 | 1.23 | 1.25
+`EtcdLearnerMode` | `false` | 1.27 | 1.29 | -
```

Comment:
Please consider keeping these in alphabetical order.

Reply:
Updated.